llvm/lib/Target/NVPTX/NVPTX.td

```diff
@@ -65,7 +65,7 @@ def : Proc<"sm_61", [SM61, PTX50]>;
 def : Proc<"sm_62", [SM62, PTX50]>;
 def : Proc<"sm_70", [SM70, PTX60]>;
 def : Proc<"sm_72", [SM72, PTX61]>;
-def : Proc<"sm_75", [SM75, PTX63]>;
+def : Proc<"sm_75", [SM75, PTX62, PTX63]>;
```

Comment:
Why are we adding PTX62 here?

According to [PTX docs](https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#release-notes-ptx-release-history) sm_75 has been introduced in PTX ISA 6.3 in CUDA-10.0.

Reply:
Yeah I wasn't sure where it should go. The docs specify it's PTX62, but I couldn't find which one that came from, so I just put it before 63. Maybe on 72?

Reply:
What are you trying to do with PTX62 feature to start with? Why do you need to add it here to start with?

In general, the features will be supplied externally. This particular place just sets the minimum required to support this particular GPU variant.

Reply:
Okay, so I can just get rid of it for this definition and it will still work? I could've just said it came with 63 and been lazy I suppose.

Reply:
I'm confused a bit here. Constraints on PTX version for GPU and for instrunctions are independent. You need both satisfied in order to use a given instruction on a given GPU.

So, to use activemask on  sm_75, you do need PTX63.
To use it on sm_52, you only need PTX62.

You do not need to change anything here. You already have correct predicates applied to the instruction itself and to the target builtin.

Reply:
Okay, so I'll remove it from the definition here and just add the PTX62. I don't have the fullest understanding of how this pTX stuff works.

Reply:
Should be fixed now, also I added the one for `nanosleep` in https://github.com/llvm/llvm-project/pull/79888.